src/daemon/src/server.rs

```diff
@@ -76,8 +76,8 @@ impl DaemonServer {
             config.read().await.batch_submission_interval_ms,
         ));
 
-        let mut monitor_interval = tokio::time::interval(Duration::from_millis(
-            config.read().await.batch_submission_interval_ms,
+        let mut process_metrics_interval = tokio::time::interval(Duration::from_millis(
```

Comment:
add the 500ms constant to get the process metrics more often, previously was related to the log submission constant (5s or 10s)